flowauth/frontend/src/UserAdminDetails.js

```diff
@@ -131,6 +138,9 @@ class UserAdminDetails extends React.Component {
         })
         .then(json => {
           onClick();
+        })
+        .catch(err => {
+          this.setState({ pageError: true, errors: err });
```

Comment:
Isn't this going to swallow _all_ possible errors from the backend, e.g. a legit 500 error? Or a 401 after the session times out?